build-wallet-electron.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# downloads the bitcoin-s-server.zip file corresponding to your operating system 
+# then sets up Suredbits Wallets dependencies
+OS="`uname`"
+case $OS in
+  'Linux')
+   OS='Linux'
+    ;;
+  'WindowsNT')
+    OS='Windows'
+    ;;
+  'Darwin') 
+    OS='Mac'
+    ;;
+  'AIX') ;;
+  *) ;;
+esac
+
+cd wallet-electron-ts
+
+Chip="`uname -m`"
+
+if [[ $Chip == 'arm64' ]]; then 
```

Comment:
What does this return on a Raspberry Pi?

Reply:
"aarch64"

Reply:
Ok so we need to modify this `if` to check "aarch64"

Reply:
Should I have it check for Raspberry pie?

Reply:
I don't know that I'm suggesting that. I just wanted to know if this would falsely id a Pi as a Mac. Most Pis should get caught on the 'Linux' check.

Reply:
Ok this is the last open conversation, is this good for our initial cut? We can always bug fix later